ibacm/src/acm.c

```diff
@@ -217,6 +217,7 @@ static const char *acme = IBACM_BIN_PATH "/ib_acme -A";
 static const char *opts_file = ACM_CONF_DIR "/" ACM_OPTS_FILE;
 static const char *addr_file = ACM_CONF_DIR "/" ACM_ADDR_FILE;
 static char log_file[128] = IBACM_LOG_FILE;
+static int umad_debug_level = 0;
```

Comment:
Looks like azure doesn't care for static initialization to zero.  I realize there is existing precedence (even in this file), but I assume we should stop initializing static variables to zero.